infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -168,10 +168,6 @@ private long initStreamLogDirectory() {
             String corfuDir = logDir.getParent().toString();
             FileStore corfuDirBackend = Files.getFileStore(Paths.get(corfuDir));
 
-            if (corfuDirBackend.isReadOnly()) {
```

Comment:
We skip this check if it is MAC os by checking OS property.

Reply:
We could but it would be better to avoid adding platform specific code as far as possible.

Reply:
If we don't do this check on Linux, and the file system is really in read-only mode. The following call new File(), it will throw an IO.readonly.exception. it is better to handle it.

Reply:
if the new File() throws an IO.readonly.exception that is also acceptable. Either case the objective of preventing a LogUnitServer from coming up would be accomplished, right?